src/System/i18n/config.ts

```diff
@@ -0,0 +1,24 @@
+import { getENV } from "Utils/getENV"
+
+// TODO: Serve static assets on the server-side
```

Comment:
@gkartalis @lidimayra what does this mean in this context?

Reply:
not 100% sure what this is referring to since I wasn't involved in the planning/implementation, but I remember some conversations for the actual translations being moved to a 3rd party like [transifex](https://www.transifex.com/) or [locize](https://locize.com/) , maybe it was something like this 🤔

Reply:
This is actually related to what was commented here: https://github.com/artsy/force/pull/9797#discussion_r840560932

We paired on it a bit, but we never managed to actually make it work, so we decided to move forward without it, as we didn't want it to block the implementation because of this.

Reply:
What's the benefit of doing that vs simply requiring?

Reply:
As I understand, this would be an optimization for the future. The server-side rendering would prevent us from loading unnecessary files if we have several JSON files for different locales.

@araujobarret might have the proper context on this.

Reply:
> this would be an optimization for the future

We should probably cross that bridge when we get there and keep it simple for now

Reply:
Presumably we'd only load this JSON in memory on the server boot — It's not going to affect performance in any meaningful way.